vars/gangplank.groovy

```diff
@@ -146,6 +147,14 @@ def _getArtifacts(params =[:], gangplankCmd) {
     return gangplankCmd
 }
 
+def _getBucket(params =[:], gangplankCmd) {
+     if (params['bucket']) {
+        gangplankCmd += "--bucket ${params['bucket']}"
```

Comment:
Is this missing a trailing space for safe concatenation in the future?

Reply:
@jlebon Welcome back from PTO =) Good catch! Thanks for checking it, here is the fix #90